addons/fire/stringtable.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Fire">
+        <Key ID="STR_ACE_Fire_category_displayName">
+            <English>ACE Fire</English>
+        </Key>
+        <Key ID="STR_ACE_Fire_Actions_PatDown">
+            <English>Pat Down Fire</English>
+        </Key>
+        <Key ID="STR_ACE_Fire_Actions_PerformingPatDown">
+            <English>Patting Down Fire</English>
+        </Key>
+        <Key ID="STR_ACE_Fire_setting_enable">
+            <English>Enable</English>
+        </Key>
+        <Key ID="STR_ACE_Fire_setting_description">
+            <English>Allow units to catch fire</English>
+        </Key>
+        <Key ID="STR_ACE_Fire_setting_flareEnable">
+            <English>Creates fire-flare at night</English>
+        </Key>
+        <Key ID="STR_ACE_Fire_setting_flareDescription">
+            <English>Enables the fire-flare effect at night</English>
```

Comment:
I don't like "Creates" in a setting title, the setting description doesn't explain what this actually does, it just repeats the title
```suggestion
            <English>Enable fire-flare at night</English>
        </Key>
        <Key ID="STR_ACE_Fire_setting_flareDescription">
            <English>Enables the fire-flare effect at night</English>
```